src/main/java/com/wavefront/sdk/common/Utils.java

```diff
@@ -261,6 +261,12 @@ public static String tracingSpanToLineData(String name, long startMillis, long d
 
   public static String spanLogsToLineData(UUID traceId, UUID spanId, @NonNull List<SpanLog> spanLogs)
       throws JsonProcessingException {
+    return spanLogsToLineData(traceId, spanId, spanLogs, null);
+  }
+
+  public static String spanLogsToLineData(UUID traceId, UUID spanId,
```

Comment:
why is this new method signature needed?

Reply:
adding `span` to the span log payload to help determine whether to sample the span logs.  this way, we can use existing logic to preprocess and evaluate the sampling decision on the `span`